content/accessibility/index.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Accessibility
+---
+
+import {Grid, Flex, Box, Link, Text, Label} from '@primer/components'
+
+
+<Grid gridTemplateColumns={['1fr', null, null, null, '1fr']} gridGap={4}>
+    
+    <div>
+        <Link href="/design/content/accessibility/accessibility-at-github.mdx" fontWeight="bold">Accessibility at GitHub</Link>
+        <Box as="p">GitHub's aim is to be the home for all developers. To be inclusive means we must consider accessibility at the core of how we design. GitHub aims for Web Content Accessibility Guidelines (WCAG) 2.1 AA compliance. This includes all of WCAG 2.0 AA plus additional considerations.</Box>
+    </div>
+    <div>
+        <Link href="/design/content/accessibility/guidelines.mdx" fontWeight="bold">Guidelines</Link>
+        <Box as="p">Basic guidelines for designers to follow when designing new or updating features.</Box>
+    </div>
+    <div>
+        <Link href="/design/content/accessibility/tools.mdx" fontWeight="bold">Tools</Link>
+        <Box as="p">A selection of tools to help product designers create accessible designs.</Box>
+    </div>
+    <div>
+        <Link href="/design/content/accessibility/focus-management.mdx" fontWeight="bold">Focus management</Link>
+        <Box as="p">Managing focus within a page or application is essential for users to successfully navigate, complete actions, and understand where they are.</Box>
+    </div>
+    <div>
+        <Link href="/design/content/accessibility/headings.mdx" fontWeight="bold">Headings</Link>
+        <Box as="p">Headings play a critical role in communicating the structure of a page. Find out why they're critical and how to create an accessible hierarchy in your pages.</Box>
+    </div>
+    <div>
+        <Link href="/design/content/accessibility/semantic-html.mdx" fontWeight="bold">Semantic HTML</Link>
+        <Box as="p">Many groups of people benefit from properly used semantic HTML. Using the correct elements allows assistive technology to accurately convey the purpose of the content to the user. Without it, they will not be able to navigate easily. Other benefits of using semantic HTML are SEO and code readability.</Box>
```

Comment:
Same comment as for the first link - make it a summary instead of guidelines.

Reply:
```suggestion
        <Box as="p">Understand when and how to use semantic HTML as you code on Github to allow for the widest user base adoption.</Box>
```